### PR TITLE
EntityManager

### PR DESCRIPTION
**Pull request by czigor**
Apart from one occurence we replace EntityManager with EntityTypeManager.

### CODE REVIEW

**Comment by bojanz** [comment]
Are the tests passing for you locally? The travis build failed (The StoreTest, to be precise).

**Comment by czigor** [comment]
No, tests are failing locally as well. Will work on it next week.

**Comment by bojanz** [comment]
Great! This is now ready. Can you please rebase & squash so that we're left with one commit?

**Comment by czigor** [comment]
Tests fail because of OrderAdminTest::testCreateOrder(). They fail in 8.x-2.x as well since the billing profile field has been made required for the order entity in 6e07c8984155800d84438084a0b82efcfbe09602.

File: modules/product/src/Tests/ProductAdminTest.php [hunk 1]
```diff
@@ -71,6 +71,56 @@ function testAddProductAdmin() {
   }
 
   /**
+   * Tests adding/removing stores to a product via the admin.
+   */
+  function testUpdateProductAdmin() {
```
Comment (bojanz) [inline modules/product/src/Tests/ProductAdminTest.php]: You are adding back a removed test by accident.

**Comment by bojanz** [comment]
Let's fix OrderAdminTest elsewhere. Remove the test and this is ready to go.

**Comment by czigor** [comment]
Thanks! I have removed the test.

**Comment by bojanz** [comment]
Thanks for the big effort, András!

**Pull request merged**